src/card/card.js

```diff
@@ -34,5 +34,6 @@ export default (data, cardStyling) => {
   } else if (attributes.extraLabel) {
     html += `<div class="org-card-text">${attributes.extraLabel}</div>`;
   }
+  html += `<div data-html="true" class="tooltip">${attributes.label || data.id}</br>${attributes.subLabel || ''}</br>${attributes.extraLabel || ''}</br>${data.measure || ''}</div>`;
```

Comment:
This will add one tooltip per card, would be better to just add a single one (appended to the divBox or something) that you then move around.

`
tooltip = divBox.append('div')
    .attr('class', 'tooltip')
    .attr('style', p => getMoreStyle(p, true))
    .on('mouseover', () => {
     tooltip.append("SOME NICE TEXT OR WHAT?")..classed('show-tooltip', true)
    }).on('mouseout'()=> {tooltip.classed(('show-tooltip', false)});
`